_examples/idv/routes.go

```diff
@@ -12,4 +12,7 @@ func initializeRoutes() {
 	router.GET("/success", showSuccessPage)
 	router.GET("/media", getMedia)
 	router.GET("/privacy-policy", showPrivacyPolicyPage)
+	/* For Testing */
+	router.GET("/digital-identity", showDigitalPage)
```

Comment:
why are these changes in idv examples?

Reply:
same temporary  test purposes and removed.